src/sql/workbench/contrib/notebook/browser/notebookThemingContribution.ts

```diff
@@ -0,0 +1,45 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IWorkbenchContribution } from 'vs/workbench/common/contributions';
+import { IConfigurationService } from 'vs/platform/configuration/common/configuration';
+import { Disposable, IDisposable } from 'vs/base/common/lifecycle';
+import { OVERRIDE_EDITOR_THEMING_SETTING } from 'sql/workbench/services/notebook/browser/notebookService';
+import { registerNotebookThemes } from 'sql/workbench/contrib/notebook/browser/notebookStyles';
+
+export class NotebookThemingContribution extends Disposable implements IWorkbenchContribution {
+	private themeParticipant?: IDisposable;
```

Comment:
I refactored the notebook theming stuff out of the service and into its own contribution.